### DISCUSSION TITLE
Amiga on Koriki OS

### DISCUSSION THREAD

**Discussion by Tekkno303** [rg35xx-cfw/Koriki#59] [General]
Is there a chance to improve the emulation of Amiga 500/600/1200/CD32 games on KorikiOS?
Amiga games run unstable (lags).
Two new improved cores have appeared on GarlicOS (PUAE and UEA4ARM). Can they be added to KorikiOS?
https://youtu.be/JIyAn1KVv24?si=68kQXotPtuNai02k
https://www.youtube.com/watch?v=iLpEwRTVgas

**Comment by acmeplus** [comment]
No new cores added yet, but puae2021 works fairly well. Download the latest interim update and test. Please report back if you see differences in performance:
https://github.com/rg35xx-cfw/Koriki/releases/tag/koriki_rg35xx_v0.9.2-test
